spyder/config/main.py

```diff
@@ -203,7 +203,7 @@
               'line_numbers': True,
               'blank_spaces': False,
               'edge_line': True,
-              'edge_line_column': 79,
+              'edge_line_columns': [79],

```

Comment:
@rlaverde i think we should not store the settings as a list but only as a string.

`'79'` or `'79,99'`

This way, the regex is used when defining the line edit, but we dont need to make check on the configuration side. We should only make these transformatios (check if it has commas... split by commas etc...) on the set_columns method

I think this way is clearer @ccordoba12 ?